Sources/SwiftProtobuf/Visitor.swift

```diff
@@ -90,9 +71,11 @@ public protocol Visitor: class {
   func visitUnknown(bytes: Data)
 }
 
-/// Forwarding default implementations of some visitor methods, for convenience.
 extension Visitor {
 
+  /// Forwards to `visitRepeatedField`. Implementors who need to handle 
```

Comment:
Does Jazzy actually pick up documentation for protocol extension methods? If so, that's great—and appears to go beyond what Xcode does. 👍